Src/cql-lm/schema/elm/library.xsd

```diff
@@ -1,171 +1,176 @@
 <?xml version="1.0"?>
 <!-- This file defines the root library element and type for the ELM schema. -->
```

Comment:
I don't think anything changed, just the indenting and some line wrapping on the documentation.